client/me/help/chat-business-concierge-notice/index.jsx

```diff
@@ -19,11 +18,14 @@ import { isBusinessPlanUser } from 'state/selectors';
 
 class ChatBusinessConciergeNotice extends Component {
 	static propTypes = {
-		translate: PropTypes.func,
-		isBusinessPlanUser: PropTypes.bool.isRequired,
 		from: PropTypes.string.isRequired,
 		selectedSite: PropTypes.object.isRequired,
 		to: PropTypes.string.isRequired,
+
+		// Connected props
+		isBusinessPlanUser: PropTypes.bool.isRequired,
+		moment: PropTypes.func,
+		translate: PropTypes.func,
 	};
 
 	static defaultProps = {
```

Comment:
This looks like something that would be present for testing, but this component doesn't appear to be tested.
With `moment` as a prop, it might make sense to remove this default. The unwrapped component isn't exported.

`moment` and `translate` could be marked as `isRequired` `propTypes` so we'd be warned of any issues.